dspace-api/src/main/java/org/dspace/app/mediafilter/MediaFilterScript.java

```diff
@@ -112,6 +115,10 @@ public void setup() throws ParseException {
             skipIds = commandLine.getOptionValues('s');
         }
 
+        if (commandLine.hasOption('d')) {
+            fromDate = LocalDate.parse(commandLine.getOptionValue('f'));
```

Comment:
@AndrewAlesik : This looks like a mistake. It's checking for option `-d` but reading a value from option `-f`

Reply:
Hi @tdonohue , thank you for attention, I've already fix it.